app/helpers/application_helper/toolbar/container_template_center.rb

```diff
@@ -0,0 +1,33 @@
+class ApplicationHelper::Toolbar::ContainerTemplateCenter < ApplicationHelper::Toolbar::Basic
+  button_group('container_template_vmdb', [
+    select(
+      :container_temolate_vmdb_choice,
+      'fa fa-cog fa-lg',
+      t = N_('Configuration'),
+      t,
+      :items => [
+        button(
+          :container_template_scan,
+          'fa fa-search fa-lg',
+          N_('Perform SmartState Analysis on this item'),
+          N_('Perform SmartState Analysis'),
+          :confirm => N_("Perform SmartState Analysis on this item?")),
+      ]
+    ),
+  ])
```

Comment:
@moolitayer SmartState Analysis Is relevant here?
Is this feature already implemented?(Would the button work do what its supposed to)